### DISCUSSION TITLE
请问这个能实现自动续订E5吗？

### DISCUSSION THREAD

**Discussion by zhujitop** [AlistGo/alist#957] [Q&A]
请问这个能实现自动续订Microsoft 365 E5使用天数吗？

**Comment by Dedicatus5457** [comment]
这个不一定，具体去看隔壁om蘑菇的回答，保险点还是去整个自动续订吧

**Reply by haohao232323** [reply]
OM的回答在哪

**Comment by Deepseaon** [comment]
基本可以的，说基本是因为没人能保证E5一定给你续，不过自从上次续订风波之后我新开的E5就只挂网盘和alist了，平时也用E5，所以连着两次反正是正常续订，我觉得没问题

**Comment by yyyytawa** [comment]
只要频繁调用E5的API(OneDrive应该可以)让微软认为你是开发者就行
